econml/inference.py

```diff
@@ -1114,10 +1132,10 @@ def percentile_point(self):
             the corresponding singleton dimensions in the output will be collapsed
             (e.g. if both are vectors, then the output of this method will also be a vector)
         """
-        lower_percentile_point = np.percentile(self.pred, (self.alpha / 2) * 100, axis=0)
-        upper_percentile_point = np.percentile(self.pred, (1 - self.alpha / 2) * 100, axis=0)
-        return np.array([lower_percentile_point]) if np.isscalar(lower_percentile_point) else lower_percentile_point, \
-            np.array([upper_percentile_point]) if np.isscalar(upper_percentile_point) else upper_percentile_point
+        alpha = self.alpha if alpha is None else alpha
+        lower_percentile_point = np.percentile(self.pred, (alpha / 2) * 100, axis=0)
+        upper_percentile_point = np.percentile(self.pred, (1 - alpha / 2) * 100, axis=0)
+        return lower_percentile_point, upper_percentile_point
```

Comment:
Is it correct that we no longer wrap in an array if the outputs are scalar?

Reply:
Yeap. was that an api we wanted to abid by?

Reply:
I'm keeping it without a numpy array. Hope that's ok. I liked it this way as it's easier to use in the cate interpreters too. Also I would epxect that someone that has a single treatment and wants to get an ATE is expecting a scalar number.